posthog/settings.py

```diff
@@ -74,9 +74,10 @@ def print_warning(warning_lines: Sequence[str]):
 else:
     JS_URL = os.getenv("JS_URL", "")
 
+DISABLE_MMDB = get_from_env("DISABLE_MMDB", False, type_cast=strtobool)  # plugin server setting disabling GeoIP feature
```

Comment:
Suggestion: default this to `TEST` and simplify below to

```
PLUGINS_PREINSTALLED_URLS: List[str] = os.getenv(
    "PLUGINS_PREINSTALLED_URLS", "https://github.com/PostHog/posthog-plugin-geoip"
).split(",") if not DISABLE_MMDB else []
```

Reply:
Right, good point!